src/tree/aksClusterTreeItem.ts

```diff
@@ -18,12 +21,11 @@ export default class AksClusterTreeItem extends AzureTreeItem implements AksClus
     constructor(
         parent: AzExtParentTreeItem,
         private readonly resource: Resource) {
-            super(parent);
+        super(parent);
 
-            // TODO: Set fancy icon for AKS cluster nodes.
-
-            this.id = this.resource.id;
-        }
+        this.iconPath = vscode.Uri.file(path.join(getExtensionPath()!, 'resources', 'aks-tools.png'));
```

Comment:
How's this look in the light theme(s)?  (You can specify different icons for light and dark themes.)